calico/felix/plugins/fiptgenerator.py

```diff
@@ -576,13 +578,29 @@ def drop_rules(self, ip_version, chain_name, rule_spec=None, comment=None,
                 "Invalid comment %r" % comment
             comment_str = '-m comment --comment "%s"' % comment
 
-        parts = [p for p in [ipt_action,
-                             chain_name,
-                             rule_spec,
-                             '--jump DROP',
-                             comment_str]
-                 if p is not None]
-        return [' '.join(parts)]
+        rules = []
+
+        if self.ACTION_ON_DROP.startswith("LOG-"):
```

Comment:
Removed the on_deny/on_allow params under my other PR.  They were just confusing; whether a particular rule should RETURN or ACCEPT, for example, is an implementation detail of the ipt generator.  It doesn't need to be passed in.